src/core/tests/TestCoreMain.py

```diff
@@ -293,6 +293,31 @@ def test_installation_operation_fail_due_to_no_telemetry(self):
         self.assertTrue(substatus_file_data[1]["status"] == Constants.STATUS_ERROR.lower())
         runtime.stop()
 
+    def test_operation_success_for_configure_patching_request(self):
+        #todo: update test with a proper patch setting file
+        pass
+        # argument_composer = ArgumentComposer()
+        # argument_composer.operation = Constants.CONFIGURE_PATCHING
+        # argument_composer.patch_mode = Constants.AUTOMATIC_BY_PLATFORM
+        # runtime = RuntimeCompositor(argument_composer.get_composed_arguments(), True, Constants.APT)
+        # runtime.set_legacy_test_type('HappyPath')
+        # CoreMain(argument_composer.get_composed_arguments())
+        #
+        # # check telemetry events
+        # self.__check_telemetry_events(runtime)
+        #
+        # # check status file
+        # with runtime.env_layer.file_system.open(runtime.execution_config.status_file_path, 'r') as file_handle:
+        #     substatus_file_data = json.load(file_handle)[0]["status"]["substatus"]
+        # self.assertEquals(len(substatus_file_data), 1)
+        # self.assertTrue(substatus_file_data[0]["name"] == Constants.CONFIGURE_PATCHING_SUMMARY)
+        # self.assertTrue(substatus_file_data[0]["status"] == Constants.STATUS_SUCCESS.lower())
+        # runtime.stop()
```

Comment:
marking todo

Reply:
Resolved. Removed todo comment and added tests for validation